File: list-00/c/exercise-02/exercise-02.c
//Calculando os segundos

#include <stdio.h>

int main(){

	float d,h,m,s,total;

	printf("Insira a quantidade de dias: ");
	scanf("%f", &d);
	printf("Insira a quantidade de horas: ");
	scanf("%f", &h);
	printf("Insira a quantidade de minutos: ");
	scanf("%f", &m);
	printf("Insira a quantidade de segundos: ");
	scanf("%f", &s);

	total = (d * 24 * 60 * 60 )  + (h * 60 * 60 ) + ( m * 60) + s;

	printf("todos estes valores em segundos equivalem a: %.2f segundos\n", total);

	return 0;
}

File: list-00/java/exercise02/exercise02.java
// Escreva um programa que leia a quantidade de dias, horas, minutos e segundos do usuário. Calcule o total em segundos.

package exercise02;
import java.util.Scanner;


public class exercise02 {
    public static void main(String[] args){
        int dias,horas,minutos,segundos;
        Scanner valores = new Scanner(System.in);
        
        System.out.println("Insira a quantidade de dias: ");
        dias = valores.nextInt();
        System.out.println("Insira a quantidade de horas: ");
        horas = valores.nextInt();
        System.out.println("Insira a quantidade de minutos: ");
        minutos = valores.nextInt();
        System.out.println("Insira a quantidade em segundos: ");
        segundos = valores.nextInt();
        
        System.out.println("A quantidade total em segundos é: "+(dias * 24 * 60 * 60) + (horas * 60 * 60) + (minutos * 60) + segundos);
        
    }
    
}
